firebase_admin/auth.py

```diff
@@ -110,23 +123,23 @@
 UserRecord = _user_mgt.UserRecord
 
 
-def _get_auth_service(app):
-    """Returns an _AuthService instance for an App.
+def _get_client(app):
+    """Returns a Client instance for an App.
```

Comment:
This seems like the sort of case where a literal isn't necessary, or maybe could be used just once.  Further references to "client instance" would work, to my ear anyway.  But it is also fine to repeat ``Client`` instance.

Reply:
Done. This is also an internal API and therefore is not included in the published docs.